add documentation for sympy.sets.fancysets.Reals

highpost opened the pull request:
<!-- Your title above should be a short description of what
was changed. Do not include the issue number in the title. -->

#### References to other Issues or PRs
<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
format, e.g. "Fixes #1234". See
https://github.com/blog/1506-closing-issues-via-pull-requests . Please also
write a comment on that issue linking back to this pull request once it is
open. -->


#### Brief description of what is fixed or changed
The sympy.sets.fancysets.Reals class has no documentation, so this is a small documentation-only PR to fill that void.


#### Other comments


#### Release Notes

<!-- Write the release notes for this release below. See
https://github.com/sympy/sympy/wiki/Writing-Release-Notes for more information
on how to write release notes. The bot will check your release notes
automatically to see if they are formatted correctly. -->

<!-- BEGIN RELEASE NOTES -->
* sets
  * Added documentation and examples for sympy.sets.fancysets.Reals.
<!-- END RELEASE NOTES -->

highpost committed:
add documentation for sympy.sets.fancysets.Reals

The sympy.sets.fancysets.Reals class has no documentation, so this is a small documentation-only PR to fill that void.

highpost committed:
minor edit to replace tabs with spaces

highpost committed:
more tab to space fixes

highpost committed:
more tab to space fixes

sylee957 reviewed (approved):
Looks good.

But I think that `pprint` can be unnecessary here

smichr commented on sympy/sets/fancysets.py:
```diff
@@ -148,7 +148,34 @@ def _boundary(self):
 
 class Reals(with_metaclass(Singleton, Interval)):
+    """
+    Represents all continuous numbers
```
Would it be good to add "finite"?

jksuom replied:
The meaning of "continuous numbers" is obscure. You probably mean "real numbers".

smichr commented on sympy/sets/fancysets.py:
```diff
@@ -148,7 +148,34 @@ def _boundary(self):
+    ========
+
+    >>> from sympy import S, Interval, Rational, pi, I, pprint
+    >>> pprint(5 in S.Reals)
```
these look the same without pprint for me -- do they look different to you?

highpost committed:
removed pprint usage

I copied the pprint usage from the Integers example, but it's clearer without it.
I'm also changing continuous numbers to real numbers.

oscarbenjamin commented:
Thanks for this!

I'll merge now
